src/styles/theme/defaultTheme.ts

```diff
@@ -425,6 +425,9 @@ export const defaultTheme = createTheme({
     },
     MuiCardHeader: {
       styleOverrides: {
+        content: {
+          display: 'inline-flex',
```

Comment:
not going to test everything, but this breaks headers layout across app, samples:

 pr
![Screenshot from 2025-06-02 13-32-14](https://github.com/user-attachments/assets/59432024-4dbf-415e-a6c1-642bd3ee2299)
![Screenshot from 2025-06-02 13-30-19](https://github.com/user-attachments/assets/ed1a8bed-320a-483c-b783-92e388311ad2)

vs master
![Screenshot from 2025-06-02 13-30-46](https://github.com/user-attachments/assets/32e11e4c-a281-4582-8ff2-e67409010d27)
![Screenshot from 2025-06-02 13-30-12](https://github.com/user-attachments/assets/dc3c7aaf-949d-41cc-85a8-f51eb67deb2e)

Reply:
OK I have completely removed fro this PR everything about the pills. So, no more checkmark, no more jumping. No more header issues, hopefully. Let's handle that in a new PR, because it is not strictly related to the vote filter layout issue.